blockchains/erc20/erc20_worker.js

```diff
@@ -24,10 +25,36 @@ class ERC20Worker extends BaseWorker {
   async init(exporter) {
     this.lastConfirmedBlock = await this.web3.eth.getBlockNumber() - constants.CONFIRMATIONS;
     setGlobalTimestampManager(exporter);
+
+    if (constants.EXPORT_BLOCKS_LIST) {
+      this.blocksList = await initBlocksList().catch((err) => { throw err; });
```

Comment:
`.catch((err) => { throw err; });`
does this make sense? as I read this - we catch to rethrow the same exception